docs/index.md

```diff
@@ -46,9 +48,8 @@ An example that uses GFMD syntax for task lists:
 ```
   </td>
   <td>&nbsp;→&nbsp;</td>
-  <td><br>
+  <td>
 <box>
-
```

Comment:
This line needs to be retained in order for the `**Things to do:**` to be rendered as markdown, see now:
![image](https://user-images.githubusercontent.com/41845017/218233424-cf2881a0-2974-4550-8a18-fa963812f760.png)
vs before:
![image](https://user-images.githubusercontent.com/41845017/218233442-36d60cfb-f257-467f-8cd4-6b52d007851c.png)

Reason: https://markbind.org/userGuide/usingHtmlJavaScriptCss.html#markdown-in-html

Reply:
Fixed in 2050515

Reply:
Have you taken a look at the preview?
![image](https://user-images.githubusercontent.com/41845017/218233947-c8b1af9d-16b6-4c58-b570-ceb806c58373.png)

Please correct me if I'm seeing things wrong: the first example has a different alignment as compared to the second example. The first one is slightly misaligned and the second one is center-aligned. Are we able to make them align in the same way?

Reply:
Ah sorry I thought that was fine based on our previous conversation. 

Computer preview: 
> ![image](https://user-images.githubusercontent.com/77204346/218265818-98159f45-a910-4ac7-b4e7-9effe9dcb6bc.png)

Mobile preview:
> ![image](https://user-images.githubusercontent.com/77204346/218265848-39880416-f488-4180-a715-435af841e82a.png)